settings.py

```diff
@@ -998,6 +998,8 @@
 DATADOG_API_KEY = None
 DATADOG_APP_KEY = None
 
+PROCESSOR_BACKEND = 'couch'  # either couch or postgres
```

Comment:
At this stage we probably don't need the toggle and can just assume that if we're not testing we should just use the Couch backend.